### DISCUSSION TITLE
Day 5 Level 3 Q1

### DISCUSSION THREAD

**Discussion by Jahnvi13** [Asabeneh/30-Days-Of-JavaScript#632] [General]
How do we find the average of the array without using loops (as it hasn't been taught yet)?

**Comment by KhuramShahzad-5335** [comment]
> How do we find the average of the array without using loops (as it hasn't been taught yet)?

//Find the average age(all items divided by number of items)
console.log(ages.length)
el1 = parseInt(ages[1]);
el2 = parseInt(ages[1]);
el3 = parseInt(ages[2]);
el4 = parseInt(ages[3]);
el5 = parseInt(ages[4]);
el6 = parseInt(ages[5]);
el7 = parseInt(ages[6]);
el8 = parseInt(ages[7]);
let average = (el1 + el2 + el3 + el4 + el5 + el6 + el7 + el8) / 2;

console.log("Average = ",average)




this is how i did that
